packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -0,0 +1,244 @@
+/* global E */
+// @ts-nocheck
+/** @file Script to replace the econ governance committee in a SwingSet Core Eval (aka big hammer) */
+
+const { details: X } = assert;
+
+const runConfig = {
+  committeeName: 'Economic Committee',
+  economicCommitteeAddresses: {
+    // gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
+    // gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+    // gov3: 'agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h',
+    gov4: 'agoric109q3uc0xt8aavne94rgd6rfeucavrx924e0ztf',
+  },
+};
+
+// #region Quasi-imports
```

Comment:
extraneous.

Reply:
I think this is saying: this section is the result of a poor-man's module system where we manually copy the functions we need from elsewhere in the SDK.

Reply:
Yes and in IDEs the whole region can be folded